.github/workflows/integration-test.yml

```diff
@@ -66,6 +66,21 @@ jobs:
           setup-script-name: setup
           admin-password: admin
 
+      # https://github.com/actions/runner-images/issues/2840#issuecomment-790492173
+      - name: Remove unnecessary files Linux
```

Comment:
There are many github actions on freeing up disk space. The description of this [action](https://github.com/marketplace/actions/free-disk-space-ubuntu) lists a few of them.